cli.js

```diff
@@ -64,11 +65,26 @@ const mds = all
 
 const groupedMds = mds.reduce(groupByPath, []);
 
+// build siteData json
+let siteData = [];
+mds
+  .map(f => {
+    // strip tags or else it doesn't seem to index correctly
+    const contentForJson = md2html(sh.cat(f)).replace(/(<([^>]+)>)/ig,"");;
```

Comment:
`md2html(sh.cat(f))` is actually quite expensive (it is dumping a file and parsing it to markdown) so we shouldn't do it twice (here and in L86).

This should happen just once, to not be wasteful.

Either above (L69) storing the parsed content into a map and then we can get the contents or somehow else.